src/Kovan-DssSpell.t.sol

```diff
@@ -594,70 +597,174 @@ contract DssSpellTest is DSTest, DSMath {
-        assertEq(flipGUSDA.wards(address(end)), 1);
-        assertEq(flipGUSDA.wards(address(flipMom)), 1);
-        //assertEq(pipGUSD.wards(address(osmMom)), 1);
-        //assertEq(pipGUSD.bud(address(spot)), 1);
-        //assertEq(pipGUSD.bud(address(end)), 1);
-        //assertEq(MedianAbstract(pipGUSD.src()).bud(address(pipGUSD)), 1);
-
-        // Join to adapter
-        assertEq(vat.gem("GUSD-A", address(this)), 0);
-        gusd.approve(address(joinGUSDA), faucetAmount);
-        joinGUSDA.join(address(this), faucetAmount);
-        assertEq(gusd.balanceOf(address(this)), 0);
-        assertEq(vat.gem("GUSD-A", address(this)), faucetAmountWad);
-
-        // Deposit collateral, generate DAI
-        assertEq(vat.dai(address(this)), 0);
-        vat.frob("GUSD-A", address(this), address(this), address(this), int(faucetAmountWad), int(100 * WAD));
-        assertEq(vat.gem("GUSD-A", address(this)), 0);
-        assertEq(vat.dai(address(this)), 100 * RAD);
-
-        // Payback DAI, withdraw collateral
-        vat.frob("GUSD-A", address(this), address(this), address(this), -int(faucetAmountWad), -int(100 * WAD));
-        assertEq(vat.gem("GUSD-A", address(this)), faucetAmountWad);
-        assertEq(vat.dai(address(this)), 0);
-
-        // Withdraw from adapter
-        joinGUSDA.exit(address(this), faucetAmount);
-        assertEq(gusd.balanceOf(address(this)), faucetAmount);
-        assertEq(vat.gem("GUSD-A", address(this)), 0);
-
-        // Generate new DAI to force a liquidation
-        gusd.approve(address(joinGUSDA), faucetAmount);
-        joinGUSDA.join(address(this), faucetAmount);
-        (,,uint256 spotV,,) = vat.ilks("GUSD-A");
-        // dart max amount of DAI
-        vat.frob("GUSD-A", address(this), address(this), address(this), int(faucetAmountWad), int(mul(faucetAmountWad, spotV) / RAY));
-        hevm.warp(now + 1);
-        jug.drip("GUSD-A");
-        // assertEq(flipGUSDA.kicks(), 0);
-        // cat.bite("GUSD-A", address(this));
-        // assertEq(flipGUSDA.kicks(), 1);
+        DSTokenAbstract(oldChief.IOU()).approve(address(oldChief), uint256(-1));
+        oldChief.free(999999999999 ether);
+        gov.approve(address(newChief), uint256(-1));
+
+        newChief.lock(80_000 ether);
+        address[] memory slate = new address[](1);
+
+        // Create spell for testing
+        SpellTestMoms spellTestMoms = new SpellTestMoms();
+
+        // System not launched, lifted address doesn't get root access
+        slate[0] = address(spellTestMoms);
+        newChief.vote(slate);
+        newChief.lift(address(spellTestMoms));
+
+        // Launch system
+        slate[0] = address(0);
+        newChief.vote(slate);
+        newChief.lift(address(0));
+        NewChiefAbstract(address(newChief)).launch();
+
+        // System launched, lifted address gets root access
+        slate[0] = address(spellTestMoms);
+        newChief.vote(slate);
+        newChief.lift(address(spellTestMoms));
```

Comment:
Should add `assertTrue(newChief.isUserRoot(address(spellTestMoms)));`

Reply:
Done!